volume/service/service.go

```diff
@@ -33,7 +33,6 @@ type VolumesService struct {
 	vs           *VolumeStore
 	ds           ds
 	pruneRunning int32
-	eventLogger  volumeEventLogger
```

Comment:
I guess it makes more sense if they both keep the `eventLogger` since they are using it. You know right away that the service sends events if you see the `eventLogger` in the struct.

Reply:
Yup, looking at it again, I think that's cleaner; updating